### PR TITLE
fake realpath, and standardize treatment of trailing / of dirs in gendoc

### PR DESCRIPTION
**Pull request by mikedanese**
No 'realpath' on mac.

@brendanburns please see if this fixes.

### CODE REVIEW

File: hack/lib/util.sh [hunk 1]
```diff
@@ -141,7 +147,7 @@ kube::util::gen-doc() {
     # Add analytics link to generated .md files
     if [[ "${file}" == *.md ]]; then
       local link path
-      path="$relative_doc_dest$file"
```
Comment (brendandburns) [inline hack/lib/util.sh]: nit: curly braces around these.

**Comment by brendandburns** [comment]
LGTM with one nit.

File: hack/lib/util.sh (hunk 1 above)
Comment (mikedanese) [inline hack/lib/util.sh]: fixed (one line below comment)

**Comment by brendandburns** [comment]
Verified on OS X.

**Pull request merged**